docs/src/main/asciidoc/qute.adoc

```diff
@@ -69,13 +71,16 @@ public class HelloResource {
     }
 }
 ----
-<1> If there is no `@ResourcePath` qualifier provided, the field name is used to locate the template. In this particular case, we're injecting a template with path `templates/hello.txt`.
-<2> `Template.data()` returns a new template instance that can be customized before the actual rendering is triggered. In this case, we put the name value under the key `name`. The data map is accessible during rendering. 
+<1> If there is no `@ResourcePath` qualifier provided, the field name is used to locate the template.
+In this particular case, we're injecting a template with path `templates/hello.txt`.
```

Comment:
Is it legal to use line breaks in source code callouts? I thought that you always need to use `+` here...

Reply:
Hm, not sure why this part was changed, maybe IntelliJ did it when I saved the doc